cypress/support/utils/post-cards.js

```diff
@@ -0,0 +1,34 @@
+const calculateClicks = (total) => {
+  const postsPerPage = 25;
+
+  // If returning the num of clicks, subtract 1 because the first page is
+  // fully populated
+  return total <= postsPerPage ? 0 : Math.ceil(total / postsPerPage) - 1;
+};
+
+const getPostCards = () => cy.get(".post-feed").find(".post-card");
+
+const loadAndSumAllPostCards = (selector) => {
+  cy.get(selector)
+    .invoke("text")
+    .then((text) => {
+      const loadMoreSelector = "[data-test-label='load-more-articles-button']";
+      const totalPosts = Number(text.trim().match(/\d+/)[0]);
+      let numOfClicks = calculateClicks(totalPosts);
```

Comment:
```suggestion
      const numOfClicks = calculateClicks(totalPosts);
```
Nitpick: this doesn't get reassigned.